syncd/syncd_notifications.cpp

```diff
@@ -78,7 +78,15 @@ void redisPutFdbEntryToAsicView(
     std::string strFdbEntry = sai_serialize_fdb_entry(fdb->fdb_entry);
 
     std::string key = ASIC_STATE_TABLE + (":" + strObjectType + ":" + strFdbEntry);
-
+	if((fdb->event_type == SAI_FDB_EVENT_AGED)||(fdb->event_type == SAI_FDB_EVENT_FLUSHED))
```

Comment:
please fix ident and add { } to each IF

Reply:
Hi Kcudnik, Thanks for review, have fixed the problems you mentioned.